src/main/kotlin/com/landoop/jdbc4/client/RestClient.kt

```diff
@@ -213,15 +213,7 @@ class RestClient(private val urls: List<String>,
   }
 
   private fun escape(url: String): String {
-    val u = URL(url)
-    val uri = URI(
-        u.protocol,
-        u.authority,
-        u.path,
-        URLEncoder.encode(u.query, "UTF-8"),
-        u.ref
-    )
-    return uri.toURL().toString().replace("%20", "+")
+    return URLEncoder.encode(url, "UTF-8").replace("%20", "+")
```

Comment:
i don't follow. we only escape the sql param content. because we escape it alone we don't have a protocol etc. so is enough. the tests say otherwise because they work now selecting from topics

Reply:
I meant, your changes yesterday + this change here just end up back to where we were before.

Reply:
Not really:
``` val escapedSql = escape(sql)
      val endpoint = "$it/api/sql/data?sql=$escapedSql"```

```val escapedSql = escape(sql)
      val endpoint = "$it/api/jdbc/insert/prepared?sql=$escapedSql"```

``` val escapedSql = escape(normalizedSql)
    val url = "${urls[0]}/api/ws/jdbc/data?sql=$escapedSql"```

These are the 3 references to escape. so it only escapes the SQL part not the entire url. the variable name is misleading